.github/workflows/build.yml

```diff
@@ -0,0 +1,294 @@
+name: build
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - pglass/crt  # TODO: for testing
```

Comment:
For now, I've limited this to just these branches as I onboard. Eventually, I think we should allow this build.yml to run for all branches. Then every pull request builds artifacts in the same way as our release pipeline (without actually publishing them anywhere. That's controlled by the list of release branches in `.release/ci.hcl`)